docs/quickstart.cn.md

```diff
@@ -108,6 +108,13 @@ Hudi还提供了获取给定提交时间戳以来已更改的记录流的功能
 如果我们需要给定提交之后的所有更改(这是常见的情况)，则无需指定结束时间。
 
 ```Java
```

Comment:
Scala?

Reply:
> Scala?

Thanks for review, all has been corrected.